### PR TITLE
some cleanup of unused python libs for bookinfo sample 

### PR DESCRIPTION
**Pull request by AndreaM12345**
**Please provide a description of this PR:**
Some of the python libs are out of date.  Using this to clean up the unused ones first and update some of the versions to more current ones

It looks like there were a change where jaeger client is no longer used and instead open telemetry is now used.  As mentioned by the git repo: https://github.com/jaegertracing/jaeger-client-python

It looks like it will be a bigger task to replace the jaeger calls in the code that should probably be in a different PR

### CODE REVIEW

**Comment by ericvn** [comment]
Looks good so far.

**Comment by ericvn** [comment]
Also related to Python is maybe what's in the build-tools image: https://github.com/istio/tools/blob/master/docker/build-tools/Dockerfile#L439C1-L477C1

**Comment by ericvn** [comment]
And have you run through the README instructions on building and testing a new set of images with these changes?

**Comment by AndreaM12345** [comment]
> And have you run through the README instructions on building and testing a new set of images with these changes?

tested and looks good

File: samples/bookinfo/README.md [hunk 1]
```diff
@@ -4,57 +4,191 @@ See <https://istio.io/docs/examples/bookinfo/>.
...
 
-The bookinfo versions are different from Istio versions since the sample should work with any version of Istio.
+Docker images are now created.
 
 ## Push docker images to docker hub
 
-After the local build is successful, you need to update the YAML file with the latest tag that you used during the build eg: `1.16.3`.
-
-Run the following script to build the docker images, push them to docker hub, and to update the YAML files in one step.
+After the local build is successful, you will need to push the images to docker hub.  You may need to login to docker before you run the command using `docker login`
```
Comment (dhawton) [inline samples/bookinfo/README.md:106]:
```suggestion
After the local build is successful, you will need to push the images to Docker Hub.  You may need to login to Docker before you run the command using `docker login`.
```

File: samples/bookinfo/README.md [hunk 2]
```diff
@@ -4,57 +4,191 @@ See <https://istio.io/docs/examples/bookinfo/>.
...
+ => [auth] user1/examples-bookinfo-reviews-v1:pull,push token for registry-1.docker.io                                                                                                            0.0s
+ => [auth] user1/examples-bookinfo-reviews-v3:pull,push token for registry-1.docker.io                                                                                                            0.0s
+ => [auth] user1/examples-bookinfo-reviews-v2:pull,push token for registry-1.docker.io                                                                                                            0.0s
++ [[ true == \t\r\u\e ]]
++ find ./samples/bookinfo/platform -name '*bookinfo*.yaml' -exec sed -i.bak 's#image:.*\(\/examples-bookinfo-.*\):.*#image: docker.io\/user1\1:test1.0#g' '{}' +
+```
+
+## Update YAML files to point to the newly created images
+
+You need to update the YAML file with the latest tag that you used during the build eg: `$HUB:$TAG`.
```
Comment (dhawton) [inline samples/bookinfo/README.md:151]:
```suggestion
You need to update the YAML file with the latest tag that you used during the build, eg: `$HUB:$TAG`.
```

**Review by dhawton — COMMENTED** [review]
Couple of nits.

**Review by ericvn — APPROVED** [review]

**Pull request merged**